recipes/quickfast/all/patches/00002-add-cmakelists.patch.txt

```diff
@@ -0,0 +1,59 @@
+--- /dev/null	2020-06-25 02:46:11.000000000 -0300
```

Comment:
Do not use it as patch, but as CMakeLists.txt directly. 

BTW Did you implement it, or it's from an external project? I see that quickfast doesn't offer cmake support.

Reply:
It is based on the one from microsoft vcpkg. I was going to write that at the beginning of the file, but I'm not sure how to put it.

The original is this one here:

https://github.com/microsoft/vcpkg/blob/9241e6c4b2ca8dbf8ec2c0997084d63bf48ce460/ports/quickfast/CMakeLists.txt

Reply:
I added a `CMakeLists.txt` with the following preamble:

https://github.com/conan-io/conan-center-index/blob/aed1f481b8344ddf80b8aa808eb23631a50c0318/recipes/quickfast/all/CMakeLists.txt#L1-L10

Also, as the patch is also theirs, there's now a mention to the origin of the patch in `conanfile.py`:

https://github.com/conan-io/conan-center-index/blob/aed1f481b8344ddf80b8aa808eb23631a50c0318/recipes/quickfast/all/conanfile.py#L47-L50

Is it enough?

Reply:
Yes, much better! Thanks a lot!